modules/nginx/manifests/redirect.pp

```diff
@@ -0,0 +1,44 @@
+define nginx::redirect (
+    $web_domain		= hiera("web_domain"),
+    $web_domain_redirect_from = hiera("web_domain_redirect_from"),
+    $ssl_crt_source     = "puppet:///modules/localconfig/ssl/*.${web_domain_redirect_from}/server.crt",
+    $ssl_key_source     = "puppet:///modules/localconfig/ssl/*.${web_domain_redirect_from}/server.key",
+    $server_name        = $name,
```

Comment:
Hmm, server_name is the same terminology used elsewhere, eg in server.pp
which sets up web_domains_external, which is why I used the same here.


On 2 April 2015 at 11:46, Branden Visser <notifications@github.com> wrote:

> In modules/nginx/manifests/redirect.pp
> <https://github.com/oaeproject/puppet-hilary/pull/135#discussion_r27647177>
> :
>
> > @@ -0,0 +1,44 @@
> > +define nginx::redirect (
> > +    $web_domain		= hiera("web_domain"),
> > +    $web_domain_redirect_from = hiera("web_domain_redirect_from"),
> > +    $ssl_crt_source     = "puppet:///modules/localconfig/ssl/*.${web_domain_redirect_from}/server.crt",
> > +    $ssl_key_source     = "puppet:///modules/localconfig/ssl/*.${web_domain_redirect_from}/server.key",
> > +    $server_name        = $name,
>
> server_name sounds more something like pp0, app1, but I think this is
> more intended to the "tenant alias" in OAE terminology. Can you rename it
> to something like tenant_alias?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/oaeproject/puppet-hilary/pull/135/files#r27647177>.
>